src/platform/datapath_raw_xdp.c

```diff
@@ -1570,6 +1560,26 @@ CxPlatXdpTx(
     }
 }
 
+_IRQL_requires_max_(DISPATCH_LEVEL)
+void
+CxPlatDpRawTxEnqueue(
+    _In_ CXPLAT_SEND_DATA* SendData,
+    _In_ uint16_t IdealProcessor
+    )
+{
+    XDP_TX_PACKET* Packet = (XDP_TX_PACKET*)SendData;
+    XDP_QUEUE* Queue = Packet->Queue;
+    CXPLAT_DBG_ASSERT(IdealProcessor == CxPlatProcCurrentNumber());
+    if (Queue->XdpWorker->ProcIndex == IdealProcessor) {
+        CxPlatListInsertTail(&Queue->WorkerTxQueue, &Packet->Link);
+        CxPlatXdpTx(Queue->XdpWorker->Xdp, Queue);
+    } else {
+        CxPlatLockAcquire(&Packet->Queue->TxLock);
+        CxPlatListInsertTail(&Queue->TxQueue, &Packet->Link);
+        CxPlatLockRelease(&Packet->Queue->TxLock);
+    }
```

Comment:
So far it does seem to be increasing latency in higher connection count tests. I am wondering if the overhead of calling the whole `CxPlatXdpTx` is too much. Let's try replacing this call with partial logic just to enqueue this packet to XDP and poke as necessary. Nothing else.

Reply:
I tried that in earlier commits.

Reply:
Yeah, [32920cf](https://github.com/microsoft/msquic/pull/2835/commits/32920cf81af6f9b93a958911fa3aca31dbeac64d)

Reply:
In the perf pipeline, the first one for this PR is only queue and poke. The second one is call the whole thing.